Vagrantfile

```diff
@@ -2,13 +2,11 @@
 # vi: set ft=ruby :
 load "settings.sh"
 
-$IP = NETWORK + ".18".to_s
-
 Vagrant.configure("2") do |config|
   
   config.vm.box = "ubuntu/bionic64"
 	
-  config.vm.network "public_network", ip: $IP, netmask: "255.255.255.0", bridge: "eno1"
+  config.vm.network "public_network", ip: LAN_IP, netmask: "255.255.255.0", bridge: "eno1"
```

Comment:
vagrant will ask user to select interface if `eno1` not found on host machine. So this also need to put in settings.sh, also `netmask`?

Reply:
Let's say that netmask is always `255.255.255.0` (only in very rare cases you may need to make it different).

About `eno1`, I think that there should be some way to discover it automatically.

Also, if there are several network interfaces, the script should be able to autodetect which one is connected to the internet (for example the wifi interface) and use it as the WAN interface of the ltsp box.

But if something cannot possibly be discovered automatically, then yes, we should put it on `settings.sh` instead of hardcoding it on the scripts.

Reply:
good point. I think the best solution could be to fetch interface name from the host machine. On first thoughts this would be helpful `VBoxManage list bridgedifs | grep -m1 Name | awk '{print $2}'` But I am sure there are better ways doing same. Will search for that.
Also should I write seperate line for `netmask` in `settings.sh`?

Reply:
> Also should I write seperate line for netmask in settings.sh?

You don't need to, since it is always fixed to `255.255.255.0`.

Reply:
> On first thoughts this would be helpful VBoxManage list bridgedifs

Try also the commands `ip address` and `ip link`.